src/Symfony/Component/HttpKernel/Tests/KernelTest.php

```diff
@@ -626,21 +626,20 @@ public function getContainerClass(): string
     }
 
     /**
-     * Returns a mock for the BundleInterface.
+     * Returns a dummy implementation for the BundleInterface.
```

Comment:
Revert? The method still returns a mock.

Reply:
Indeed, thanks!